### ISSUE TITLE
Customised reports - tab in Nutil

### ISSUE THREAD

**Issue by sharoncy** [Neural-Systems-at-UIO/nutil#160]

**Comment by sharoncy** [comment]
Nicolaas and I had a meeting with Ingvild to discuss ideas for this. Ingvild said that it would be useful with a customised excel sheet with data in the right format for generating graphs. The graphs could either be generated automatically with Nutil or generated with a python script. We discussed 2D to 3D extrapolations of counts, but decided that it is best not to implement this in Nutil as extrapolations are only valid as long as there is sufficient sampling, etc. The user would need a good understanding of the algorithms, sampling frequency, region sizes, etc, to generate valid results.

**Comment by sharoncy** [comment]
@ingvildeb requested an update on this on 18.05.21. No progress to date and no plans for this anytime soon as there are other priorities. But once we have capacity again I agree this is a great idea.

**Comment by ingvildeb** [comment]
It would also be great to have the object reports include the average object size for each brain region (and custom region). This information is needed when making corrected calculations of densities / total numbers.

**Comment by Majpuc** [comment]
Is this taking care of by the new analysis python scripts?

**Comment by sharoncy** [comment]
No, this is not covered by the python scripts. I'll do testing of the object reports when Nutil is finalized: check what's in place and if anything can be improved.
